app/schema.py

```diff
@@ -131,3 +131,32 @@ class Feedback(BaseModel):
 class Resource(BaseModel):
     name: constr(max_length=255)
     item_id: constr(max_length=255)
+
+
+class Comments(BaseModel):
```

Comment:
Add a docstring: this is comparable to our "Reviews" collection in MongoDB. (These are comments that a mentor or mentee can make at any point in time.) And include the relations that are indicated by arrows between tables on the milanote chart, ie "note_id references note_id in Notes collection"